wgpu/resources/commit-sha

```diff
@@ -0,0 +1 @@
+v0.5dev
```

Comment:
This is going to be the git tag for the release that we've downloaded right? Can we add that in a comment too somewhere, plz?

Reply:
Well, there's two files.. `commit-sha` is extracted from the wgpu-native zipfile and we mostly include it to have full traceability. Our script works with the other file `wgpu_native-version` to track which release we downloaded. It's kind of a coincidence that they match. I'm not sure if we can assume that will always be the case?

Reply:
Makes sense!